app/Livewire/Dashboard.php

```
@@ -16,15 +19,85 @@ class Dashboard extends Component
+    public function unpinPage(string $url): void
+    {
+        $user = auth()->user();
+        $team = $user?->currentTeam();
+
+        if (! $team) {
+            return;
+        }
+
+        // Rate limit: 10 pins per minute per user
+        $key = 'toggle-pin:'.$user->id;
+        if (RateLimiter::tooManyAttempts($key, 10)) {
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\RateLimiter' in method 'unpinPage'. (undefined)

(StaticAccess)